templates/includes/docs_versions.html

```diff
@@ -0,0 +1,9 @@
+<section class="documentation__versions">
+  <h4>Previous versions</h4>
+
+  <ul>
+    <li><a href="http://maas.ubuntu.com/docs1.9/">1.9</a></li>
```

Comment:
Please target classes here instead of the raw element. For example: `documentation__versions-item`